packages/run-protocol/src/runStake/attestation.js

```diff
@@ -206,8 +206,8 @@ const makeAttestationIssuerKit = async (zcf, stakeBrand, lienBridge) => {
  * Authorize each account holder to lien some of their staked assets
  * and get an attestation that the lien is in place.
  *
- * @param {ZCF} zcf
- * @param {Brand} stakeBrand
+ * @param {ZCF<import('./runStake').RunStakeTerms>} zcf
```

Comment:
`attestation.js` isn't supposed to know about / rely on `runStake.js`. Only the other way around. If this type was somehow necessary, something went wrong.